src/containers/Home/styles/HomeStyles.js

```diff
@@ -10,21 +10,20 @@ const style = StyleSheet.create({
     flex: 1,
     padding: 10,
     alignContent: 'flex-start',
-    backgroundColor: colors.DarkPrimary
+    backgroundColor: colors.DarkPrimary,
   },
   homeTitle: {
     fontSize: 20,
     paddingVertical: 10,
-    color:'#FFFFFF'
+    color: '#FFFFFF',
   },
-  homeTextStyle:{
-    color:'#FFFFFF',
-    fontWeight:'bold'
+  homeTextStyle: {
+    color: '#FFFFFF',
```

Comment:
I notice that you use constants for colors, here you could also use a constant for the color black.